### ISSUE TITLE
Documentation error for adding submodule?

### ISSUE THREAD

**Issue by twelve17** [OpenWatch/FFmpegWrapper#3]
I attempted to add the submodule per the read me:

  `git submodule add Submodules/FFmpegWrapper https://github.com/OpenWatch/FFmpegWrapper.git`

However, that yielded this error:

   `repo URL: 'Submodules/FFmpegWrapper' must be absolute or begin with ./|../`

Instead, I was able to do:

```
  cd <mymodule_dir>
  mkdir Submodules
  git submodule add https://github.com/OpenWatch/FFmpegWrapper.git Submodules/FFmpegWrapper
```

I'm using git version 1.8.1.

**Comment by mmehdi** [comment]
you need to write like this:
`git submodule add https://github.com/OpenWatch/FFmpegWrapper.git Submodules/FFmpegWrapper `

**Comment by twelve17** [comment]
@mmehdi yep, that is what I ended up doing (not sure if you saw, I had noted that in my OP)
